app/scripts/modules/core/src/pipeline/executions/execution/executionMarker.less

```diff
@@ -35,4 +35,10 @@
   &.requires-attention {
     background-color: var(--color-alert);
   }
+  i {
+    color: #fff;
```

Comment:
Is there an existing color var we can use instead?

Reply:
sorry yes bad copy paste